.github/workflows/erlang.yml

```diff
@@ -10,17 +10,23 @@ jobs:
 
     strategy:
       matrix:
-        otp: [21.3, 22.3, 23.3]
+        otp: [21, 22, 23, 24]
```

Comment:
`setup-beam` will choose the highest possible version in the <x> range (just check the https://github.com/AdRoll/rebar3_format/pull/275/checks).

Reply:
Comment makes no sense any more.